sources/packages/backend/workflow/test/2021-2022/constants/program-year.constants.ts

```diff
@@ -1,6 +0,0 @@
-export const PROGRAM_YEAR = "2021-2022";
```

Comment:
Not a part of this PR. I totally missed in last PR, is there a reason to retain only bpmn files of 2021-2022 ?

![image](https://github.com/bcgov/SIMS/assets/54600590/96657cd4-6def-41f6-aaff-69f08027a864)

Reply:
The 2021-2022 was not removed from the SIMS, only the E2E tests for now to avoid extra time while executing the tests. We have another ticket to remove the 2021-2022 program year (#2995).

Reply:
Got it. the search didn't return the form definition which made me think it is left. But I got it now.